### PR TITLE
Update Package-Restore.md

### PR DESCRIPTION
**Pull request by bergano65**
Полезная информация для внесения предложений.
1. Изучите [краткие руководства по стилю для локализации](https://docs.microsoft.com/globalization/localization/styleguides), в которых приведены **10 наиболее важных правил** из руководства Microsoft по стилю.
2. Перейдите на [языковой портал Microsoft](https://www.microsoft.com/language), на котором содержатся **стандартизированные переводы терминов** для всех продуктов Microsoft.

### CODE REVIEW

**Comment by olprod** [comment]
The PR is not being built because the target branch `live` is filtered in `.openpublishing.publish.config.json` on `live` branch.

File: docs/consume-packages/Package-Restore.md [hunk 1]
```diff
@@ -40,43 +40,43 @@ ms.locfileid: "79428441"
...
-- восстановите пакеты автоматически ИЛИ
-
+- восстановите пакеты автоматически
+или
 - восстановите пакеты вручную.
 
 ### <a name="restore-packages-automatically-using-visual-studio"></a>Автоматическое восстановление пакетов с помощью Visual Studio
 
-Восстановление пакетов осуществляется автоматически при создании проекта на основе шаблона или во время сборки проекта. При этом учитываются значения параметров, представленных в разделе [Включение и отключение восстановления пакетов](#enable-and-disable-package-restore-in-visual-studio). В NuGet 4.0 и более поздних версий также доступно автоматическое восстановление при изменении проекта типа пакета SDK (как правило, проекта .NET Core или .NET Standard).
+ÄВосстановление пакетов осуществляется автоматически при создании проекта на основе шаблона или во время сборки проекта. При этом учитываются значения параметров, представленных в разделе [Включение и отключение восстановления пакетов](#enable-and-disable-package-restore-in-visual-studio). В NuGet 4.0 и более поздних версий также доступно автоматическое восстановление при изменении проекта типа пакета SDK (как правило, проекта .NET Core или .NET Standard).
```
Comment (bergano65) [inline docs/consume-packages/Package-Restore.md:61]: sorry my mistake

**Comment by olprod** [comment]
The PR is not being built because the target branch `live` is filtered in `.openpublishing.publish.config.json` on `live` branch.

**Comment by olprod** [comment]
The PR is not being built because the target branch `live` is filtered in `.openpublishing.publish.config.json` on `live` branch.

**Comment by olprod** [comment]
The PR is not being built because the target branch `live` is filtered in `.openpublishing.publish.config.json` on `live` branch.

**Pull request merged**